oreClient/src/main/assets/api.js

```diff
@@ -17,7 +17,10 @@ export class API {
                     contentType: isFormData ? false : 'application/json',
                     data: isBodyRequest && !isFormData ? JSON.stringify(data) : data,
                     processData: !(isFormData || isBodyRequest),
-                    headers: {'Authorization': 'OreApi session=' + session}
+                    headers: {
+                        'Authorization': 'OreApi session=' + session,
+                        'Csrf-Token': csrf
```

Comment:
What is this here for?

Reply:
If you do something which is not GET its needed. Before this was added globally but now I do Ajax requests on discourse which doesn't accept the header.